### PR TITLE
Added app documentation and status for testrun

### PR DESCRIPTION
**Pull request by luismmontielg**
@mposchen do not merge yet, because it will update live docs, and the code is not on production yet, we will merge this when the other code is live

### CODE REVIEW

File: rest.rst [hunk 1]
```diff
@@ -100,6 +100,84 @@ You can decide how to handle errors in your code based on the HTTP status code.
...
+Curl example
+************
+
+.. code-block:: console
+
+    curl -X POST \
+        -H 'Accept: application/vnd.testmunk.v1+json' \
+        -H 'Content-Type: application/json' \
+        -d '{"appName":"My-new-project"}' \
+        "http://AQS0LCTvCv6mTwod5PwtU2i1JVY2J6rW@staging.testmunk.com/api/apps"
```
Comment (mposchen) [inline rest.rst]: @luismmontielg staging.testmunk.com ?

File: rest.rst [hunk 2]
```diff
@@ -211,7 +289,7 @@ Curl Example
 Selecting Devices to Test On
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To select devices to test on, go to testmunk.com and navigate to ``Account Settings`` > ``REST API``.
+To select devices to test on, go to testmunk.com and navigate to ``Account Settings`` > ``REST API``. Or you can also set the devices the moment you create a Testrun using the `Create a new testrun`_ endpoint.
```
Comment (mposchen) [inline rest.rst]: @luismmontielg don't capitalize Testrun -> testrun

**Comment by mposchen** [comment]
@luismmontielg see my comments. @fryclukasz I want you to be the "quality gatekeeper" for our documentation. So have it on your todo to review and test these pushes and raise your voice if something is not clear or we should add further explanation.

File: rest.rst (hunk 2 above)
Comment (luismmontielg) [inline rest.rst]: @mposchen ok, I just pushed the change

**Comment by fryclukasz** [comment]
@mposchen ok

**Pull request merged**